components/main_layout/templates/head.jade

```diff
@@ -18,7 +18,7 @@ script( src=asset('/assets/scroll_frame.js') )
 
 //- Set viewport size depending on if the page is responsive or not
 if sd.IS_RESPONSIVE && (userAgent && !userAgent.match('iPad'))
-  meta( name='viewport', content='width=device-width, initial-scale=1' )
+  meta( name='viewport', content='width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no' )
```

Comment:
i checked out a few pages, but it didn't seem offensive for all `IS_RESPONSIVE` pages to be non-scalable. Lmk if you have other thoughts @craigspaeth 

another option is to set an IS_MOBILE flag using something from [here](https://github.com/artsy/force/blob/master/lib/middleware/redirect_mobile.coffee#L16)